pyproject.toml

```diff
@@ -70,7 +71,15 @@ source = [
 skip_empty = true
 
 [tool.pytest.ini_options]
-addopts = "-vv --tb=auto --disable-warnings"
+addopts = "-v --tb=short --disable-warnings -ra --no-header --show-capture=log"
```

Comment:
add a description to this configuration and why it's added. Also drop `--disable-warnings`.

Reply:
Sure. I'm doing all this logging-related stuff because I use the terminal to run pytest. With default settings, the output is too cluttered with unnecessary information, making it painful to read.

- vv was producing output too verbose with lots of unneeded information for testing purposes only
- tb=short makes tracebacks for failed tests more concise: prints the failing line + related context instead of the whole damn function
- ra prints a **R**ecap at **the end** of the output. `a` stands for all tests except passed ones.
- no-header reduces verbosity at the beginning of the output with little helpful info
- show-capture=log This is because pytest, by default, captures EVERYTHING. Usually, this is not an issue until we work with libraries (spark) that produce A LOT of output which often we don't need for unit testing. This is captured and printed. On top of that, logging written by us gets printed twice, once in the captured `stderr/out`, once in the `captured logs`. The end result is that pytest output is very cluttered. By capturing `log` only, it's much cleaner without missing out useful info (especially logging of our own).